docs/features/default.test.md

```diff
@@ -0,0 +1,173 @@
+﻿# General concerns
+- [ ] Parsing: `default < e1 > e2`
```

Comment:
The regular default is `default(T)`, not `default<T>`, so I don't think there will be ambiguity.